rasa/engine/recipes/default_recipe.py

```diff
@@ -519,16 +598,14 @@ def _add_core_train_nodes(
                 policy_with_end_to_end_support_used or requires_end_to_end_data
             )
 
+            needs = {}
+            if requires_end_to_end_data:
+                needs["precomputations"] = "end_to_end_features_provider"
+            needs.update(self._get_needs_from_args(component.clazz, "train"))
+            # during core training we use a stripped down version of the domain
+            needs["domain"] = "domain_for_core_training_provider"
             train_nodes[f"train_{component_name}{idx}"] = SchemaNode(
-                needs={
-                    "training_trackers": "training_tracker_provider",
```

Comment:
I assume the training_trackers will be picked up as part of the needs_from_args?

Reply:
yes